pyomo/contrib/mindtpy/tests/test_mindtpy_global.py

```diff
@@ -11,11 +11,9 @@
 from pyomo.environ import *
 from pyomo.opt import TerminationCondition
 
-required_solvers = ('baron', 'cplex_persistent')
+required_solvers = ('ipopt', 'cplex_persistent')
```

Comment:
Do you not need a global solver for these tests?

Reply:
I think the Pyomo servers don't have `baron` installed, so these tests are skipped previously, lowering the code coverage. For these selected models, `ipopt` is enough to find a global optimal solution.